src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/LocalsTests.cs

```diff
@@ -3492,6 +3492,7 @@ void M(string[] args)
 {
   // Code size        7 (0x7)
   .maxstack  1
+  .locals init (int V_0)
```

Comment:
I'm assuming these are harmless? They're present in most of the other tests in this file - I'm not sure why my change triggered this.

Reply:
I suspect what happened here is that you transformed `x + y + z` into something with a temporary variable, which then got dropped (i.e. left unused) in a subsequent optimization/transformation.  It is a bit worrying, but not incorrect.

---
In reply to: [275783989](https://github.com/dotnet/roslyn/pull/35006#discussion_r275783989) [](ancestors = 275783989)